### ISSUE TITLE
Relax package.json restrictions on OS (and maybe architecture)

### ISSUE THREAD

**Issue by voutilad** [AnatoliyGatt/deep-freeze-node#5]
As far as I can tell, there's no need to restrict to just macOS, Windows, and Linux since the library is purely javascript. I ran into this when installing an npm package on an OpenBSD machine and that package failed installation due to depending on `deep-freeze-node`.

I figured a full pull request is overkill, so here are the offending lines:

https://github.com/AnatoliyGatt/deep-freeze-node/blob/28c24b59934003074c221badb9e26a5e69844dee/package.json#L55-L56

Removing them on my x86-64 OpenBSD system and running `npm run test` shows tests passing.

**Comment by mojavelinux** [comment]
Yes, please. This is very abnormal for a Node package. It's causing installation problems for some of my users after switching to deep-freeze-node. There's no reason a freeze helper should be restricting installation of a Node application.

I've submitted a PR to have these restrictions removed. Thanks in advance!
